### ISSUE TITLE
Dlmread Error in `dautoalign4warp` on Desktop Ubuntu

### ISSUE THREAD

**Issue by YuX-Ren** [alisterburt/autoalign_dynamo#10]
Hi,
I’ve encountered the same issue as described in [Dlmread error with autoalign_dynamo in matlab #1](https://github.com/alisterburt/autoalign_dynamo/issues/1#issue-937980893), but on desktop Ubuntu instead of Windows. Could you help investigate the cause of this error?

### Command:
```bash
dautoalign4warp('./imod', 1.35, 10, 85.3, './dynamo_alignments')
```

### Error Log:
```
Output checking is active for this step.
Fitter has changed its traceSet
Attention: 35 tilts do not contain any markers.
Reference tilt moved to 15 (18.35 degrees)
Multilevel computing
Scanning level 1: 37 samples, interval 5.00, from -90.00 to 90.00
... best psi for level 1: 0.00
Scanning level 2: 5 samples, interval 2.50, from -5.00 to 5.00
... best psi for level 2: 0.00
Scanning level 3: 11 samples, interval 0.50, from -2.50 to 2.50
... best psi for level 3: 0.00
Scanning level 4: 11 samples, interval 0.10, from -0.50 to 0.50
... best psi for level 4: 0.00
------------------------------------------------------------
Entering reindexing step 1
A total of 1 different labels found in column 20 ("tomo"):
label number 1: 1 particles.
[table_merge] Warning: single table remains. Merging not necessary.
Merging 1 tables
Done merging tables
resulting table: 1 x 35
Thinning with threshold: 20.00
Initial number of markers: 1
Marker number post-thinning: 1
Multilevel computing
Scanning level 1: 37 samples, interval 5.00, from -90.00 to 90.00
... best psi for level 1: 0.00
Scanning level 2: 5 samples, interval 2.50, from -5.00 to 5.00
... best psi for level 2: 0.00
Scanning level 3: 11 samples, interval 0.50, from -2.50 to 2.50
... best psi for level 3: 0.00
Scanning level 4: 11 samples, interval 0.10, from -0.50 to 0.50
... best psi for level 4: 0.00
Warning: Dimensions of arrays being concatenated are not consistent.

In autoalign_warn (line 7)
In handle_exception (line 10)
In dautoalign4warp (line 32)
Warning: Attempting to continue...
In autoalign_warn (line 7)
In handle_exception (line 11)
In dautoalign4warp (line 32)
Updated RMSD stats for experimental bead positions vs. projection model
min: 0.00
max: 0.00
mean: 0.00
sdev: 0.00
No new tilt series found at 14:37:54

2 tilt series processed in this session so far
Going to sleep for 30s...
No new tilt series found at 14:38:24
```

### Additional Concerns:
The warning mentions:  
**"35 tilts do not contain any markers."**  
In my understanding, this suggests there are 35 tilts without gold fiducials. However, I am testing on `TS-01` of `empair-10164`, which should contain multiple gold fiducials in every tilt. Could this be related to the issue?

**Comment by alisterburt** [comment]
Errors finding beads are usually due to incorrect bead sizes being used - did you export the motion corrected images at 1.35A/px?

![image](https://github.com/user-attachments/assets/ed0bd6c3-0bc8-483a-a98c-5417a259dff6)

For info, this workflow was put together ~4 years ago and is not actively supported now, all of the packages used have changed significantly. I still think dynamos fiducial based tilt series alignment is the best but would generally advise using WarpTools for processing right now

https://warpem.com/user_guide/warptools/quick_start_warptools_tilt_series/

**Comment by YuX-Ren** [comment]
Thanks for your advice. However, the link you provided can not be opened. I have tried the warp tools (https://warpem.github.io/warp/user_guide/warptools/quick_start_warptools_tilt_series/#running-m-with-mcore) and dynamo built-in alignment(https://www.sciencedirect.com/science/article/pii/S0969212624002417). It just worked fine😊.
